### PR TITLE
Reuse SchemaExport in CreateSchema/DropSchema in tests

### PR DESCRIPTION
**Pull request by bahusoid**

### CODE REVIEW

File: src/NHibernate.Test/TestCase.cs [hunk 1]
```diff
@@ -48,6 +49,8 @@ protected virtual string MappingsAssembly
 			get { return "NHibernate.DomainModel"; }
 		}
 
+		protected SchemaExport SchemaExport => _schemaExport ?? (_schemaExport = new SchemaExport(cfg));
```
Comment (fredericDelaporte) [inline src/NHibernate.Test/TestCase.cs]:
```
NHSpecificTest\NH2195\SQLiteMultiCriteriaTest.cs(35,18): warning CS0108: 'SQLiteMultiCriteriaTest.SchemaExport(Configuration)' hides inherited member 'TestCase.SchemaExport'. Use the new keyword if hiding was intended.
```

Release build will fail. Maybe is it time to enable warning as error in the test project debug config too.

**Comment by bahusoid** [comment]
Can I also push here some small modification in `SchemaExport` class to avoid unnecessary calls for `SchemaExport.Initialize` for each `dropSQL`/`createSQL` statement (private Execute method doesn't need to call Initialize)?

**Comment by fredericDelaporte** [comment]
So it would be just removing this [line](https://github.com/nhibernate/nhibernate-core/blob/5.0.3/src/NHibernate/Tool/hbm2ddl/SchemaExport.cs#L170), because this method is called only in one other method which already call `Initialize`.

Still a bit unrelated, but well why not.

**Comment by bahusoid** [comment]
@fredericDelaporte Yeah. Also renamed it to ExecuteInitialized to differentiate it from other public Execute methods where Initialize is called.

**Pull request merged**